resources/src/commands/onboard_user.rs

```diff
@@ -0,0 +1,32 @@
+use crate::commands::{chat_post_message, conversations_open};
+use serde_json::{json, Value};
+
+// TODO: write docs
+pub async fn run(body: &Value) {
+    let username: &str = body["event"]["user"]["id"]
+        .as_str()
+        .unwrap_or("invalid_user_name");
+    let first_name: &str = body["event"]["user"]["profile"]["first_name"]
+        .as_str()
+        .unwrap_or("");
+    let text: String = format!(":codedevils_flash: Hey welcome to CodeDevils {} :codedevils_flash:. I am DevilBot and I don't know much yet, but here's what I do know: \
+        This Slack workspace serves as the main communication platform for all things CodeDevils :partywizard:. All our announcements can be found in the <#C30L07P18> channel. \
+        This includes all meetings and meeting recordings! I'd like you to go to the <#CMGU8033K> channel and introduce yourself. After that, come on over to\
+        <#C2N5P84BD>. Most of my creators are there all day.", &first_name);
+
+    // After acceptance testing change this to the username
+    let conversations_open_json = json!({ "users": "WGJRL9NJD" });
```

Comment:
What does this variable name mean? What conversations were opened?

Reply:
The conversation gets opened between the user name passed in and the owner of the API key (so devilbot) passed in from above (In the new env var that you will need to add before deploying). Of course more users can be passed through that users param. Something more descriptive would be `json string to pass to conversations open` Next would be to change this "WGJRL9NJD" to the user name that gets passed in after taken from the request body.